content/rooms/v11.md

```diff
@@ -0,0 +1,284 @@
+---
+title: Room Version 11
+type: docs
+weight: 100
+---
+
+This room version builds on [version 10](/rooms/v10) while clarifying redaction
+rules.
+
+## Client considerations
+
+### Redactions
+
+{{% added-in this=true %}} The top-level `origin`, `membership`, and `prev_state` properties
```

Comment:
This isn't rendering right: the changes are in a separate paragraph to the "[New in this version]" label.

I think you want:

```suggestion
{{< added-in this=true >}} The top-level `origin`, `membership`, and `prev_state` properties
```

(cf https://gohugo.io/content-management/shortcodes/#shortcodes-without-markdown)

Reply:
ditto in v9, if you fancy fixing that while you're here

Reply:
I fixed it for v6, v9, and v11.